app/v2/coinswap/internal/keeper/querier.go

```diff
@@ -12,12 +11,9 @@ import (
 func NewQuerier(k Keeper) sdk.Querier {
 	return func(ctx sdk.Context, path []string, req abci.RequestQuery) (res []byte, err sdk.Error) {
 		switch path[0] {
-		case types.QueryLiquidity:
+		case types.QueryLiquidities:
```

Comment:
It seems to only query one liquidity.

Reply:
Yeah, it does, it is also used here: https://github.com/irisnet/irishub/pull/1866/files#diff-f1c5e0ca5c61ad1bf8f5ae81620b7816R30
Just wanna keep it same as the api path `/coinswap/liquidities`
Anyway, I think we do need to revert this to `QueryLiquidity` in case we need a `query liquidities by conditions` someday. Thx